packages/dev/core/src/Misc/rgbdTextureTools.ts

```diff
@@ -55,12 +54,16 @@ export class RGBDTextureTools {
             const shaderLanguage = isWebGPU ? ShaderLanguage.WGSL : ShaderLanguage.GLSL;
             internalTexture.isReady = false;
 
-            if (!this._ShaderImported) {
-                this._ShaderImported = true;
+            if (!this._EngineResourcesImported) {
+                this._EngineResourcesImported = true;
                 if (isWebGPU) {
-                    await Promise.all([import("../ShadersWGSL/rgbdDecode.fragment"), import("../ShadersWGSL/rgbdEncode.fragment")]);
+                    await Promise.all([
+                        import("../ShadersWGSL/rgbdDecode.fragment"),
+                        import("../ShadersWGSL/rgbdEncode.fragment"),
+                        import("../Engines/WebGPU/Extensions/engine.renderTarget"),

```

Comment:
For this one, I'm 80% sure that I saw it sometimes work and sometimes not work with this, so I suspect there are multiple things that need it, and one of them imports it, so perhaps depending on order of async operations it may or may not work.